shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Val.hs

```diff
@@ -77,3 +84,52 @@ instance Val Coin where
   coin = id
   inject = id
   size _ = 1
+
+-- ============================================
+-- Constants needed to compute size and size-scaling operation
+
+-- address hash length is always same as Policy ID length
+addrHashLen :: Integer
```

Comment:
These various definitions are used only within the `scaledMinDeposit` function. So they should be moved to a `where` clause under that function rather than defined a the top level